SKILL.tsv

```diff
@@ -4756,37 +4756,37 @@ SKILL_20150717_004755	Range attack rate increases by 3 while [Finestra] is activ
 SKILL_20150717_004770	$Enemies hit by [Fireball] will receive flame damage for 5 seconds with a 10% chance per attribute level. The flame damage is based on the character's INT.
 SKILL_20150717_004771	$Increases the Fire property damage for 5 seconds by 5 per attribute level, when an ally steps on [Fire Wall].
 SKILL_20150717_004772	{memo X}When an enemy is near [Enchant Fire] range, the enemy's Fire resistance will decrease by 10% for 5 secs.
-SKILL_20150717_004773	$Enemies hit by [Magnetic Force] have a 5% chance to become stunned for 4 seconds per attribute level.
+SKILL_20150717_004773	$Enemies hit by [Magnetic Force] can become stunned for 4 seconds by 5% chance per attribute level.
 SKILL_20150717_004774	$Enemies hit by [Hangman's Knot] will receive additional damage equal to 20% magic attack per attribute level.
 SKILL_20150717_004775	$Hangman's Knot: Splash Defense
 SKILL_20150717_004776	$Decreases the AoE Defense Ratio of the enemies gathered through [Hangman's Knot] by 1.
 SKILL_20150717_004777	{memo X}Attack increase effect of [Swell left Arm] Increase by 1 per Attribute level.
 SKILL_20150717_004778	{memo X}Defense and Attack increase effect of [Swell Right Arm] increases by 1 per attribute level.
 SKILL_20150717_004779	{memo X}Intelligence increase effect of [Swell Brain] increases by 1 per attribute level.
-SKILL_20150717_004780	The enemy's movement speed will decrease by 25% whose size got bigger by [Swell Body] and Physical and Magical ATK rate will increase by 20% per Attribute level.
-SKILL_20150717_004781	The enemy's movement speed will increase by 20% whose size got smaller by [Shrink Body] and Physical and Magical ATK rate will decrease by 25% per Attribute level.
-SKILL_20150717_004782	Whenever the size of an enemy changes by [Swell Body], more damage will be incurred which is 20% of Magical ATK rate per Attribute level.
-SKILL_20150717_004783	Whenever the size of an enemy changes by [Shrink Body], more damage will be incurred which is 20% of Magical ATK rate per Attribute level.
-SKILL_20150717_004784	When you attack an enemy that is in [Swell Body] when [Left Arm Enhancement] is being activated, the weakened ATK rate status will continue. Duration will increase by 8 secs per Attribute level and [Weakened ATK] can be overlapped max 10 times.
-SKILL_20150717_004785	$When they are hit by [Hail], there is a 5% chance to inflict [Freezing] for 5 seconds per attribute level.
+SKILL_20150717_004780	Enemies affected by [Swell Body] will have their movement speed decreased by 25% and Physical and Magical ATK rate increased by 20% per attribute level.
+SKILL_20150717_004781	Enemies affected by [Shrink Body] will have their movement speed increased by 20% and Physical and Magical ATK rate decreased by 25% per attribute level.
+SKILL_20150717_004782	Enemies affected by [Swell Body] will take more damage, 20% Magical ATK per attribute level.
```

Comment:
Deals 20% damage proportional to 20% magic attack per attribute level. and enemies affected by [Swell Body].